index.html

```diff
@@ -58,7 +58,207 @@ <h4>LET'S CONNECT</h4>
+                    <img src="images/Snapshoot Portfolio.png" alt="">
+                </div>
+                <h2 class="colorh2">Tonic</h2>
+                <div>
+                    <div class="canopy">
+                        <h4 class="colorh4">CANOPY</h4>
+                        <ul>
+                            <li> 
+                                <span>
+                                    <img width="8px"  height="8px" src="images/Ellipse 12.png" alt="">
+                                </span> Back End Dev
+                            </li>
+                            <li> 
+                                <span>
+                                    <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
+                                </span> 2015
+                            </li>
+                        </ul>
+                    </div>
+                    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li class="languages">HTML</li>
+                        <li class="languages">CSS</li>
+                        <li class="languages">JavaScript</li>
+                    </ul>
+                    <div class="seeproject">
+                        <a title="#" href="#">See project</a>
+                    </div>
+                </div>
+            </article>
+            <article class="sections">
+                <div class="scrnshot">
+                    <img src="images/Snapshoot Portfolio (3).png" alt="">
+                </div>
+                <h2 class="colorh2">Multi-Post Stories</h2>
+                <div>
+                    <div class="canopy">
+                        <h4 class="colorh4">CANOPY</h4>
+                        <ul>
+                            <li> 
+                                <span>
+                                    <img width="8px"  height="8px" src="images/Ellipse 12.png" alt="">
+                                </span> Back End Dev
+                            </li>
+                            <li> 
+                                <span>
+                                    <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
+                                </span> 2015
+                            </li>
+                        </ul>
+                    </div>
+                    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li class="languages">HTML</li>
+                        <li class="languages">CSS</li>
+                        <li class="languages">JavaScript</li>
+                    </ul>
+                    <div class="seeproject">
+                        <a title="#" href="#">See project</a>
+                    </div>
+                </div>
+            </article>
+        </section>
+        <section id="aboutme">
```

Comment:
Please remember to implement the border-top-right-radius as the previous reviewer ask you to make the card `About me` to looks like the one on the Figma design 🥇